### DISCUSSION TITLE
input signal in audio interface but no signal in jamulus

### DISCUSSION THREAD

**Discussion by KlaasJK** [jamulussoftware/jamulus#2125] [Technical Support & Help] [answered]
hi all, 
I'm new to jamulus, have a macbook pro with a behringer UMC204HD audio interface and an semi-acoustic bass guitar. 
I can see my signal coming in on the Behringer, clipping if i add gain,  but no signal appearing in my jamulus screen. 
(also not in servers, obviously) 
Audio settings on the mac: sound-in is set to UMC204 HD
Settings in jamulus: In: UMC204HD
What am I missing? 

Many thanks for answers,
Klaas

**Comment by dcorson-ticino-com** [comment]
Hi Klaas!
Are you connected to a server?
That is the first step to get sound.
You hear what comes back from the server, if not connected to a server, no sound.
Let us know if connecting works.
Don

**Comment by KlaasJK** [comment]
Hi

Thanks for your response!
I was connected to a server, did hear the other players, they also saw me
(I heard them mention it) but my sound didn't come through

On Wed, Nov 17, 2021 at 7:56 PM DonC ***@***.***> wrote:

> Hi Klaas!
> Are you connected to a server?
> That is the first step to get sound.
> You hear what comes back from the server, if not connected to a server, no
> sound.
> Let us know if connecting works.
> Don
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jamulussoftware/jamulus/discussions/2125#discussioncomment-1660271>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWRNJJ3ST4QKXJRC62QIEYDUMP3EXANCNFSM5IHRUMJQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
klaas kuitenbrouwer - www.otocron.net

**Reply by dcorson-ticino-com** [reply]
Take a look here https://jamulus.io/wiki/Client-Troubleshooting
I assume you have selected your interface in the audio settings.
Maybe someone with a Mac can chime in on possible Mac oriented things.

**Reply by MarkusZehner** [reply]
Hi, 
had the same problem today, just a guess, but did you enable jamulus to acces the microphone within the security settings?
Cheers!

**Comment by KlaasJK** [comment, marked as answer]
sorry for slow reply, but yes, that was the problem.

On Sun, Nov 21, 2021 at 4:03 PM Markus Zehner ***@***.***>
wrote:

> Hi,
> had the same problem today, just a guess, but did you enable jamulus to
> acces the microphone within the security settings?
> Cheers!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jamulussoftware/jamulus/discussions/2125#discussioncomment-1677536>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWRNJJ4T32FZSQCRXGAWBUDUNEC4XANCNFSM5IHRUMJQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
klaas kuitenbrouwer - www.otocron.net

**Reply by henkdegroot** [reply]
Glad to see your feedback that you have resolved the issue. Perhaps you can mark this as "answered" as well?

**Comment by KlaasJK** [comment]
thank you, yes will do now!

On Wed, Dec 1, 2021 at 8:10 AM Henk De Groot ***@***.***>
wrote:

> Glad to see your feedback that you have resolved the issue. Perhaps you
> can mark this as "answered" as well?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jamulussoftware/jamulus/discussions/2125#discussioncomment-1726523>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWRNJJ7EE5BS675F72LYPALUOXC5DANCNFSM5IHRUMJQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
klaas kuitenbrouwer - www.otocron.net
